usr/share/rear/prep/Linux-i386/370_detect_secure_boot.sh

```diff
@@ -0,0 +1,34 @@
+test "$SECURE_BOOT_BOOTLOADER" && return 0
+
+# Note:
+# this is more of a hack than a really good solution.
+# The good solution would check the EFI variables for the bootloader
+# that is used to boot the system. But this is not implemented yet for
+# secure boot.
+#
+# The code in usr/share/rear/rescue/default/850_save_sysfs_uefi_vars.sh
+# could be used as a starting point for this, however currently
+# it tries to read the actual boot loader# used only as a last resort
+# if well-known files are not found.
+#
+if type -p mokutil ; then
```

Comment:
Only a personal coding style question:
Why not
```
# Nothing to do when there is no 'mokutil':
type -p mokutil || return 0
```
cf. "Return early, return often" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
I wanted to write a debug line about missing `mokutil` to help us with "secure boot auto-config didn't work" support cases.